telegraf.js

```diff
@@ -190,7 +190,6 @@ class Telegraf extends Composer {
         console.error('Failed to process updates.', err)
         this.polling.started = false
         this.polling.offset = 0
-        this.polling.stopCallback && this.polling.stopCallback()
```

Comment:
And where is the second time?

Reply:
See the issue for details.
https://github.com/telegraf/telegraf/blob/1273631bd17e22d144526bca213f64ea347f4dec/telegraf.js

Reply:
What issue?

Reply:
> Fixes #1043 bug

> Linked issues
https://github.com/telegraf/telegraf/issues/1043